### ISSUE TITLE
cargo install sozu error

### ISSUE THREAD

**Issue by tsingson** [sozu-proxy/sozu#851]
error msg
```
 Compiling clap v4.0.29
error[E0004]: non-exhaustive patterns: `rustls::ProtocolVersion::DTLSv1_3` not covered
    --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/sozu-lib-0.14.1/src/https_rustls/session.rs:1397:11
     |
1397 |     match version {
     |           ^^^^^^^ pattern `rustls::ProtocolVersion::DTLSv1_3` not covered
     |
note: `rustls::ProtocolVersion` defined here
    --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/rustls-0.20.7/src/enums.rs:20:9
     |
5    | / enum_builder! {
6    | |     /// The `ProtocolVersion` TLS protocol enum.  Values in this enum are taken
7    | |     /// from the various RFCs covering TLS, and are listed by IANA.
8    | |     /// The `Unknown` item is used when processing unrecognised ordinals.
...    |
20   | |         DTLSv1_3 => 0xFEFC
     | |         ^^^^^^^^ not covered
21   | |     }
22   | | }
     | |_-
     = note: the matched value is of type `rustls::ProtocolVersion`
help: ensure that all possible cases are being handled by adding a match arm with a wildcard pattern or an explicit pattern as shown
     |
1406 ~         ProtocolVersion::Unknown(_) => "tls.version.Unknown",
1407 ~         rustls::ProtocolVersion::DTLSv1_3 => todo!(),
     |

   Compiling termion v1.5.6
   Compiling tempfile v3.3.0
   Compiling paw v1.0.0
```
build in centOS 8

**Comment by tsingson** [comment]
build from git source success.

**Comment by Keksoj** [comment]
Same issue as #835 , see the issue for details.

This will be fixed soon with a `0.14.2`, shipped with a fix.
